icons.md

```diff
@@ -3,7 +3,7 @@
 |   Android   |   android   | <img src="https://user-images.githubusercontent.com/65664185/138502465-89cfadf2-6b54-4f3d-ac44-ceacdd4824ba.png" width=65% height=30%>                       |     Apple      |     apple      | <img src="https://user-images.githubusercontent.com/65664185/138502540-8e9b80bf-deae-4566-a41a-c63623e83c21.png" width=100% height=30%> |
 |    Book     |    book     | <img src="https://user-images.githubusercontent.com/76985777/145391108-f8c08f8e-679f-45a3-ad58-83ef60aa28fe.png" width=65% height=30%>                       |   Codeforces   |   codeforces   | <img src="https://user-images.githubusercontent.com/91655303/148160942-870fdbb4-a57c-4861-afaa-241835390645.png" width=100% height=30%> |
 |  Codewars   |  codewars   | <img src="https://user-images.githubusercontent.com/76241195/147094572-962f911f-9ebd-4615-b72f-ff076f4033e8.png"  width=65% height=30%>                      |     DEV.to     |     dev.to     | <img src="https://user-images.githubusercontent.com/76241195/147094431-b194a188-0a63-4ce3-89c9-1d7578405318.png" width=100% height=30%> |
-|   Discord   |   discord   | <img src="https://user-images.githubusercontent.com/65664185/138502295-d82d98cf-2b42-4926-ab68-e45b2cfe8605.png" width=65% height=30%>                       |     Dollar     |     dollar     | <img src="https://user-images.githubusercontent.com/76985777/145393429-c03c5c3e-0416-4d28-be21-8a01a0c3dff5.png" width=100% height=30%> |
+|   Discord   |   discord   | <img src="https://assets-global.website-files.com/6257adef93867e50d84d30e2/625e5f775305fb9733389e0f_Discord-Logo-Black%20(1).png" width=65% height=30%>                       |     Dollar     |     dollar     | <img src="https://user-images.githubusercontent.com/76985777/145393429-c03c5c3e-0416-4d28-be21-8a01a0c3dff5.png" width=100% height=30%> |
```

Comment:
This is a more black version of the icon. 🙂
```suggestion
|   Discord   |   discord   | <img src="https://user-images.githubusercontent.com/51391473/187728370-34aac260-8fea-4d41-aaeb-06b1643ccd9a.png" width=50% height=30%>                       |     Dollar     |     dollar     | <img src="https://user-images.githubusercontent.com/76985777/145393429-c03c5c3e-0416-4d28-be21-8a01a0c3dff5.png" width=100% height=30%> |
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient